about.html

```diff
@@ -0,0 +1,65 @@
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" >
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" >
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" >
+    <title>Elearnig Website 2022, 2023</title>
+    <link rel="stylesheet" href="./style.css" >
+    <script defer src="./script.js"></script>
+    <script src="https://kit.fontawesome.com/10729b3e67.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css">
+  </head>
+  <body class="about-body">
+      <header>
+        <div class="top-header">
+          <figure class="humburger fa-beat">
+            <span  class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+           </figure>
+           <a href="#" id="exit" class="nullifier"><i class="fa-solid fa-xmark fa-2xl fa-beat"></i></a>
+          <ul>
+            <li><a href="#"><i class="fa-brands fa-facebook-f fa-lg"></i></a></li>
+            <li><a href="#"></a><i class="fa-brands fa-twitter fa-lg"></i></li>
+            <li>English</li>
+            <li>MyPage</li>
+            <li>Layout</li>
+          </ul>
+         </div>
+         <div class="sec-header">
+          <div class="logo">
+            <a href="./index.html"> <img src="./images/logo.png" alt="logo for e learning website" width="100" height="50"></a>
+          </div>
+          <nav class="nav">
+           <ul class="toggle">
+                <li><a href="./index.html" class="">Home</a> </li>
+                <li><a href="#" class="">Program</a> </li>
+                <li><a href="#" class="">Courses</a> </li>
+                <li><a href="#" class="start-learning" id="start-learning">Start Learning</a></li>
+             </ul>
```

Comment:
- [ ] [OPTIONAL] Kindly make sure to use the same navbar and footer for both the `home` page and the `about` page, make sure that the navbar has a 100% width

- You're about page navbar
![Screenshot_20221203_102629](https://user-images.githubusercontent.com/25227022/205433974-1e698724-0dd2-424b-90f3-a1f992ed7f4d.png)